packages/vuepress-theme-odyssey/styles/components/_OdoSidebar.scss

```diff
@@ -0,0 +1,51 @@
+.odo-sidebar {
+  padding: $spacing-l $spacing-m 0 $spacing-l;
+
+  @include mq(m) {
+    display: none;
+  }
+
+  .odo-site--title,
+  .odo-site--title:visited {
+    display: block;
+    padding: $spacing-xs;
+    color: var(--odo-text-heading, #{$text-heading});
+    font-size: $size-title-4;
+    font-weight: 600;
+    text-decoration: none;
+  }
+
+  .odo-sidebar--content {
+    display: grid;
```

Comment:
Could we move this up to `.odo-sidebar` itself and use Grid to lay out all of the children?